tests/functional/clients/standalone/package_list/fixtures/alpine.py

```diff
@@ -55,8 +55,8 @@
         "type": "APKG",
         "name": "alpine-baselayout",
         "cpes": [
-            "cpe:2.3:a:*:alpine-baselayout:3.2.0-r6:*:*:*:*:*:*:*",
             "cpe:2.3:a:alpine-baselayout:alpine-baselayout:3.2.0-r6:*:*:*:*:*:*:*",
+            "cpe:2.3:a:*:alpine-baselayout:3.2.0-r6:*:*:*:*:*:*:*",
```

Comment:
Are these tests/fixtures order sensitive?

Reply:
they are, which is important for the matching process in grype (we should be trying the most specific CPE first). We could make the order agnostic, but I think this will be more important to assert order in future releases.